app/src/features/projects/view/ProjectHeader.tsx

```diff
@@ -234,7 +226,11 @@ const ProjectHeader = () => {
                 aria-controls="projectSettingsMenu"
                 aria-haspopup="true"
                 style={{ marginLeft: '0.5rem' }}
-                onClick={handleClick}>
+                onClick={handleClick}
+                disabled={
+                  !hasProjectRole([PROJECT_ROLE.PROJECT_EDITOR, PROJECT_ROLE.PROJECT_LEAD]) &&
```

Comment:
![image](https://user-images.githubusercontent.com/35311998/232875288-36407cb0-b33c-4b6e-a9b9-7d6b502be3dd.png)

This button should be hidden from the viewer not disabled